### PR TITLE
Mangaz : fix getting pictures

### PR DESCRIPTION
**Pull request by MikeZeDev**
they switched from encrypting to scrambling

### CODE REVIEW

File: web/src/engine/websites/Mangaz.ts [hunk 1]
```diff
@@ -4,15 +4,24 @@ import { Chapter, DecoratableMangaScraper, Manga, Page, type MangaPlugin } from
...
-type ImgObj = {
-    img: string[],
-    b: {
-        Enc: {
-            key: string,
-            iv: string
-        }
-    }
+type ImageData = {
+    img: string,
```
Comment (ronny1982) [inline web/src/engine/websites/Mangaz.ts:10]: Consider more appropriate field name e.g., `imagePath`

File: web/src/engine/websites/Mangaz.ts [hunk 2]
```diff
@@ -59,39 +68,31 @@ export default class extends DecoratableMangaScraper {
...
-        let decrypted = await crypto.subtle.decrypt({
-            name: 'AES-CBC',
-            iv: iv,
-        }, secretKey, encrypted,);
-
-        const sdecrypted = new TextDecoder('utf-8').decode(decrypted);
-        decrypted = Uint8Array.from(window.atob(sdecrypted), char => char.charCodeAt(0));
-        return await Common.GetTypedData(decrypted);
+        const blob = await Common.FetchImageAjax.call(this, page, priority, signal);
+        const scrambleData = page.Parameters as ScrambleData;
```
Comment (ronny1982) [inline web/src/engine/websites/Mangaz.ts:89]: Optional typing is now available for page parameters and should be used when possible
See: https://github.com/manga-download/haruneko/commit/abb147fdbf7079781a5e4e44f6c83ccf3f1b3aa6

File: web/src/engine/websites/Mangaz_e2e.ts [hunk 3]
```diff
@@ -16,9 +16,9 @@ const config: Config = {
         title: '13'
     },
     entry: {
-        index: 0,
-        size: 525_231,
-        type: 'image/jpeg'
+        index: 1,
+        size: 1_402_344,
```
Comment (ronny1982) [inline web/src/engine/websites/Mangaz_e2e.ts:20]: Any smaller image available ?

**Review by ronny1982 — APPROVED** [review]

File: web/src/engine/websites/Mangaz.ts (hunk 2 above)
Comment (MikeZeDev) [reply web/src/engine/websites/Mangaz.ts:89]: Nice !

File: web/src/engine/websites/Mangaz.ts (hunk 1 above)
Comment (ronny1982) [reply web/src/engine/websites/Mangaz.ts:10]: I was referring to the `img` field, the typename itself was appropriate 😅

**Pull request merged**